javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/FunctionBodyFeatures.qll

```diff
@@ -127,6 +127,18 @@ ASTNode getAnASTNodeWithAFeature(Function f) {
   result = getAnASTNodeToFeaturize(f)
 }
 
+int getNumCharsInFunction(Function f) {
+  result =
+    strictsum(ASTNode node | node = getAnASTNodeWithAFeature(f) | getTokenizedAstNode(node).length())
+}
+
+// Evaluator string limit is 5395415 characters. We choose a limit lower than this.
+private int getMaxChars() { result = 1000000 }
+
+Function getFeaturizableFunction(Function f) {
```

Comment:
The structure of this predicate is a bit confusing.  I suggest converting to a `predicate isFunctionFeaturizable(Function f)` predicate instead, or (perhaps better) just inlining it.

Reply:
Will inline.